ts/test/test-profile-serializer.ts

```diff
@@ -19,36 +19,46 @@ import {serializeHeapProfile, serializeTimeProfile} from '../src/profilers/profi
 import {TimeProfile, TimeProfileNode} from '../src/v8-types';
 
 import {anonymousFunctionHeapProfile, anonymousFunctionTimeProfile, heapProfile, timeProfile, v8AnonymousFunctionHeapProfile, v8AnonymousFunctionTimeProfile, v8HeapProfile, v8TimeProfile} from './profiles-for-tests';
+import {copy} from './test-helpers';
 
 const assert = require('assert');
 
 describe('serializeTimeProfile', () => {
   let dateStub: sinon.SinonStub;
+  let memoryUsageStub: sinon.SinonStub;
+
   before(() => {
     dateStub = sinon.stub(Date, 'now').returns(0);
+    memoryUsageStub = sinon.stub(process, 'memoryUsage').returns({
+      external: 1024,
+      rss: 2048,
+      heapTotal: 4096,
+      heapUse: 2048,
+    });
   });
   after(() => {
     dateStub.restore();
   });
   it('should produce expected profile', () => {
-    const timeProfileOut = serializeTimeProfile(v8TimeProfile, 1000);
+    const timeProfileOut = serializeTimeProfile(copy(v8TimeProfile), 1000);
```

Comment:
For my education, why do need to pass a clone to the `serializeTimeProfile` function? Does it mutate its argument?

Reply:
It's not necessary -- removed this.